### PR TITLE
fixing bug

### PR DESCRIPTION
**Pull request by Macbull**
Worker model don't have owner attribute. we need a new permission for this.

### CODE REVIEW

File: crowdsourcing/viewsets/worker.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ class WorkerViewSet(viewsets.ModelViewSet):
     queryset = Worker.objects.all()
     serializer_class = WorkerSerializer
     lookup_field = 'profile'
-    permission_classes = [IsOwnerOrReadOnly]
+    # permission_classes = [IsOwnerOrReadOnly]
```
Comment (dmorina) [inline crowdsourcing/viewsets/worker.py]: because?

Comment (Macbull) [inline crowdsourcing/viewsets/worker.py]: Worker model does not have any 'owner' attribute

Comment (dmorina) [inline crowdsourcing/viewsets/worker.py]: ok, i see, then either this permission class needs to be updated OR the user can only update his own data

Comment (dmorina) [inline crowdsourcing/viewsets/worker.py]: merging for now

**Pull request merged**